README.md

```diff
@@ -0,0 +1,77 @@
+# Covid-19 data traker
+
+> SPA web app that helps to get information about Covid-19
```

Comment:
- [ ] Add a more meaningful project description explaining all the necessary information about the app.

Reply:
DONE ✅